widlparser/constructs.py

```diff
@@ -1103,41 +1110,41 @@ def __repr__(self):
         return output + '[argumentlist: ' + (repr(self.arguments) if (self.arguments) else '') + ']]'
 
 
-class ImplementsStatement(Construct):  # [ExtendedAttributes] identifier "implements" identifier ";"
+class IncludesStatement(Construct):  # [ExtendedAttributes] identifier "includes" identifier ";"

```

Comment:
1. We should retain support for `implements` statements for now, until every spec has been migrated over to `includes`.
2. The production [*IncludesStatement*](https://heycam.github.io/webidl/#prod-IncludesStatement) does not contain a leading `[ExtendedAttributes]`.